tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc

```diff
@@ -295,34 +294,6 @@ TEST_F(EnergyPlusFixture, ZoneTempPredictorCorrector_CorrectZoneHumRatTest)
     Zone(1).IsControlled = true;
     CorrectZoneHumRat(1);
     EXPECT_NEAR(0.008, Node(5).HumRat, 0.00001);
-
-    // Deallocate everything
```

Comment:
Cleaned out a bunch of old deallocates and other inits that are already covered in `clear_state` functions.